tests/optim/utils/test_model_utils.py

```diff
@@ -9,6 +9,20 @@
 from tests.helpers.basic import BaseTest, assertTensorAlmostEqual
 
 
+class TestPadReflectiveA4D(BaseTest):
+    def test_pad_reflective_a4d(self) -> None:
+        x = torch.ones(1, 3, 4, 4)
+        padding = [2] * 8
+        x_out = model_utils.pad_reflective_a4d(x, padding)
+        self.assertEqual(list(x_out.shape), [4, 7, 8, 8])

```

Comment:
why is the dimension [4, 7, 8, 8] here ? 
If we compare it with the constant equivalent then we get: torch.Size([5, 7, 8, 8]). 1 + 2 + 2 for the batch size dimension
```
torch.nn.functional.pad(x, padding, "constant", 0).shape
```